src/util.ts

```diff
@@ -334,11 +337,47 @@ export const generateAppSecret = (
  *     `v` value.
  * 2.  Newer transaction types such as `FeeMarketEIP1559Transaction` will subtract
  *     27 from the `v` that gets passed in, so we need to add `27` to create `initV`
- * @param tx - An @ethereumjs/tx Transaction object
+ * @param tx - An @ethereumjs/tx Transaction object or Buffer (serialized tx)
  * @param resp - response from Lattice. Can be either legacy or generic signing variety
+ * @returns bn.js BN object containing the `v` param
  */
 export const getV = function (tx, resp) {
-  const hash = tx.getMessageToSign(true);
+  let chainId, hash, type;
+  const txIsBuf = Buffer.isBuffer(tx);
+  if (txIsBuf) {
+    hash = Buffer.from(keccak256(tx), 'hex');
+    try {
+      const legacyTxArray = rlpDecode(tx);
+      if (legacyTxArray.length === 6) {
+        // Six item array means this is a pre-EIP155 transaction
+        chainId = null;
+      } else {
+        // Otherwise the `v` param is the `chainId`
+        chainId = new BN(legacyTxArray[6]);
+      }
+      // Legacy tx = type 0
+      type = 0;
+    } catch (err) {
+      // This is likely a typed transaction
+      try {
+        const txObj = EthTxFactory.fromSerializedData(tx);
+        type = txObj._type;
```

Comment:
```suggestion
        //@ts-expect-error -- Accessing private property
        type = txObj._type;
```

Fixes a type error preventing build.